bin/lib-gm

```diff
@@ -500,6 +506,31 @@ get_chain_id() {
   fi
 }
 
+# Returns empty string if --long is not supported
+get_sdk_version() {
+  GAIAD_BINARY="$(get_gaiad_binary "$1")"
+  RESULT="$("$GAIAD_BINARY" version --long 2>&1 | sed -n '/cosmos_sdk_version/ s/cosmos_sdk_version: v\(.*\)$/\1/p')"
+  echo "$RESULT"
+}
+
+get_genesis_cmd_prefix() {
+    GENESIS_BREAKING_VERSION="0.47.0"
+    SDK_VERSION=$(get_sdk_version "$1")
```

Comment:
I know it's nitpicking but shellcheck would also complain:

```suggestion
    SDK_VERSION="$(get_sdk_version "$1")"
```

Reply:
Approved my own suggestion so we can move to merging. Apologies. I think it was minor enough issue that we'd agree on the resolution.